generators/entity-client/templates/vue/src/main/webapp/app/entities/entity.component.ts.ejs

```diff
@@ -3,15 +3,14 @@ import { mixins } from 'vue-class-component';
 import { Component, Inject } from 'vue-property-decorator';
 import Vue2Filters from 'vue2-filters';
 import { I<%= entityAngularName %> } from '@/shared/model/<%= entityModelFileName %>.model';
-import AlertService from '@/shared/alert/alert.service';
+import AlertMixin from '@/shared/alert/alert.mixin';
```

Comment:
I think you need to apply this changes to entity-update.component.ts.ejs file too

Reply:
Actually, there is no need to do that. In the entity-update.component.ts.ejs we call the `AlertService` to push a new alert. 

The mixin I introduce only get rid of the code used to display an alert by fetching the values in the store, you'll still have to inject the `AlertService` in some component to push a new one.

Reply:
Ok you're right!